migrations/20220801135839_answer_count_and_timestamps.js

```diff
@@ -28,7 +28,7 @@ exports.down = async function (knex) {
   });
 
   await knex.schema.table("answers", (table) => {
-    table.dropColumn("created_at");
+    table.dropColumn("timestamp");
```

Comment:
`created_at` and `timestamp` are two distinct concepts so they should both exist. Timestamp is the timestamp of the bitcoin transaction, whereas created_at is the time at which this record was inserted into the SQL database.

Reply:
maybe `timestamp` should belong to the onchain-event table then, and `created_at` is okay in `questions` and `answers` tables